### DISCUSSION TITLE
Node to Deno rust .js

### DISCUSSION THREAD

**Discussion by HaskellRustian** [tauri-apps/tauri#2572] [Ideas]
What do you think of the functioning of Tauri with deno?

**Comment by balupton** [comment]
It's the roadmap item that I'm waiting for before I start experimenting with Tauri in my hobby projects. As then Tauri will not just be better than Electron, but significantly surpass it.

**Reply by Cassielxd** [reply]
https://gitee.com/stringlxd/tauri-cc

**Reply by Cassielxd** [reply]
https://github.com/Cassielxd/tauri-cc
